optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/tenant/Tenant.java

```diff
@@ -56,23 +56,29 @@
     @OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL, optional = false)
     private TenantConfiguration configuration;
 
+    @OneToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL, optional = false)
+    private RosterState rosterState;
```

Comment:
Can you create a jira to remove TenantConfiguraiton and RosterState from the Tenant class (to cut down the eager fetching cost when fetching all tenants and to reduce the transaction span) and give them their own CRUD methods in the REST TenantService?

Reply:
https://issues.jboss.org/browse/PLANNER-1050